<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1228</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="2" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Logs and about&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="toolTipDuration">
       <number>2</number>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;LOG&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="toolTip">
        <string extracomment="Browser">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Browse here!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="toolTipDuration">
        <number>11</number>
       </property>
       <attribute name="icon">
        <iconset>
         <normaloff>../../../usr/share/icons/Papirus-Dark/16x16/apps/browser.svg</normaloff>../../../usr/share/icons/Papirus-Dark/16x16/apps/browser.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Browser</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QWebEngineView" name="webEngineView">
          <property name="url">
           <url>
            <string>https://wiki.archlinux.org/</string>
           </url>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="logs">
       <property name="toolTip">
        <string>Logs and About</string>
       </property>
       <property name="toolTipDuration">
        <number>2</number>
       </property>
       <attribute name="icon">
        <iconset>
         <normaloff>../../../usr/share/icons/Papirus-Dark/16x16/categories/setroubleshoot_icon.svg</normaloff>../../../usr/share/icons/Papirus-Dark/16x16/categories/setroubleshoot_icon.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Logs n' System</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <widget class="QTextBrowser" name="textBrowser">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Logs:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QCalendarWidget" name="calendarWidget">
          <property name="gridVisible">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QCalendarWidget::SingleSelection</enum>
          </property>
          <property name="horizontalHeaderFormat">
           <enum>QCalendarWidget::ShortDayNames</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QTextBrowser" name="textBrowser_3">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;PCI Devices (lspci):&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QTextBrowser" name="textBrowser_2">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;CPU info (lscpu):&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../usr/share/icons/Papirus-Dark/16x16/actions/go-next.svg</normaloff>../../../usr/share/icons/Papirus-Dark/16x16/actions/go-next.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="gobtn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../usr/share/icons/Papirus-Dark/16x16/actions/go-today.svg</normaloff>../../../usr/share/icons/Papirus-Dark/16x16/actions/go-today.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../usr/share/icons/Papirus-Dark/16x16/actions/reload.svg</normaloff>../../../usr/share/icons/Papirus-Dark/16x16/actions/reload.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="font">
          <font>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../usr/share/icons/Papirus-Dark/16x16/actions/back.svg</normaloff>../../../usr/share/icons/Papirus-Dark/16x16/actions/back.svg</iconset>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLineEdit" name="urledit">
         <property name="text">
          <string>http://</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLCDNumber" name="lcdNumber">
         <property name="smallDecimalPoint">
          <bool>false</bool>
         </property>
         <property name="digitCount">
          <number>3</number>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1228</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>webEngineView</sender>
   <signal>titleChanged(QString)</signal>
   <receiver>textBrowser</receiver>
   <slot>append(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>webEngineView</sender>
   <signal>urlChanged(QUrl)</signal>
   <receiver>textBrowser</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>783</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>webEngineView</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>webEngineView</receiver>
   <slot>reload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>released()</signal>
   <receiver>webEngineView</receiver>
   <slot>forward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>webEngineView</sender>
   <signal>loadProgress(int)</signal>
   <receiver>progressBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>616</x>
     <y>664</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>724</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>webEngineView</sender>
   <signal>loadFinished(bool)</signal>
   <receiver>progressBar</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>706</x>
     <y>670</y>
    </hint>
    <hint type="destinationlabel">
     <x>706</x>
     <y>720</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>webEngineView</sender>
   <signal>loadProgress(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
